src/reducers/issues.reducer.js

```diff
@@ -61,7 +64,9 @@ export default handleActions(
     [SET_SORTING]: (state, { payload }) =>
       flow([set('sorting', payload.sorting), set('currentPage', 1)])(state),
     [RESET_SORTING]: state =>
-      flow([set('issuesState', null), set('sorting', null)])(state)
+      flow([set('issuesState', null), set('sorting', null)])(state),
```

Comment:
whenever you are reseting something in the reducer you can use initialState.<property you want to reset> and it will make it more clear IMO